devops/vaults.env

```diff
@@ -16,8 +16,8 @@ VUE_APP_AUTH_API_URL="op://API/$APP_ENV/auth-api/AUTH_API_URL"
 VUE_APP_AUTH_API_VERSION="op://API/$APP_ENV/auth-api/AUTH_API_VERSION"
 VUE_APP_LEGAL_API_URL="op://API/$APP_ENV/legal-api-le/LEGAL_API_URL"
 VUE_APP_LEGAL_API_VERSION_2="op://API/$APP_ENV/legal-api-le/LEGAL_API_VERSION_2"
-VUE_APP_NAICS_API_URL="op://API/$APP_ENV/naics-api/NAICS_API_URL"
-VUE_APP_NAICS_API_VERSION_2="op://API/$APP_ENV/naics-api/NAICS_API_VERSION"
+VUE_APP_NAICS_API_URL="op://API/$APP_ENV/naics-api-le/NAICS_API_URL"
+VUE_APP_NAICS_API_VERSION_2="op://API/$APP_ENV/naics-api-le/NAICS_API_VERSION"
```

Comment:
I created this new section in the API vaul in 1Password (for Dev + Test + Prod).

Reply:
Should've went to this PR first 😆 Got my answer 👍